platforms/corda-ent/charts/bridge/values.yaml

```diff
@@ -0,0 +1,119 @@
+# Default values for bridge.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+deployment:
+  annotations: {}
+
+# Provide the name of the node.
+# e.g. nodeName: bridge
+nodeName: bridge
+
+# Provide information regarding the CENM Networkmap.
+# Used to curl the Networkmap service.
+nmap:
```

Comment:
this should be the URL instead, as nmap will be in another domain/k8s cluster.

Reply:
Removed from value file in [ec7e5a7](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/897/commits/ec7e5a75829cd0887cdedb55ee0c60e19266150b). This was left over from old experimentation to get network-parameters from the Networkmap service. It will now be generated in node-initial-registration and then put in Vault.